### PR TITLE
doc: release notes: ARC

### PR DESCRIPTION
**Pull request by ruuddw**
Adds 2.4 release notes for ARC.

Signed-off-by: Ruud Derwig <Ruud.Derwig@synopsys.com>

### CODE REVIEW

**Review by abrodkin — APPROVED** [review]
Overall look good but just one tiny comment: MW tools are being spelled differently all the time :)
1. MetaWare Toolchain
2. MetaWare toolchain
3. ARC MetaWare toolchain

FWIW if we look here https://www.synopsys.com/dw/ipdir.php?ds=sw_metaware it says "DesignWare ARC MetaWare Development Toolkit", not sure though we want to go that far, so maybe just use "ARC MetaWare toolchain" everywhere?

File: doc/releases/release-notes-2.4.rst [hunk 1]
```diff
@@ -230,13 +237,26 @@ Boards & SoC Support
...
 
   * ARM Cortex-M1/M3 DesignStart FPGA
+  * ARC QEMU support for EM and HS family
 
 * Made these changes in other SoC series:
 
   * STM32L4/STM32WB: Added support for Low Power Mode
   * STM32H7/STM32WB/STM32MP1: Added Dual Core concurrent register access
     protection using HSEM
+  * Increase cpu frequency for ARC nsim_hs_smp
```
Comment (MaureenHelm) [inline doc/releases/release-notes-2.4.rst:247]: Please use past tense here and throughout

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Please fixup the commit messages to address the compliance check failure

**Comment by MaureenHelm** [comment]
Please rebase

**Comment by ruuddw** [comment]
@mhelm 
> Please rebase

Done (twice by now :-)

**Review by MaureenHelm — APPROVED** [review]
> > Please rebase
> 
> Done (twice by now :-)

Thank you!

**Pull request merged**